godump.go

```diff
@@ -360,7 +381,7 @@ func formatByteSliceAsHexDump(b []byte, indent int) string {
 	return sb.String()
 }
 
-// callerLocation returns the file and line number of the caller at the specified skip level.
+// callerLocation returns the file and line number of the caller at the specified skippedStackFrames level.
 func callerLocation(skip int) (string, int) {
```

Comment:
Caller location is not used, I'm removing it with #28

- #28

Please avoid updating it, it would help to avoid conflicts